### PR TITLE
zxing-cpp: dependency version update of stb

### PR DESCRIPTION
**Pull request by SerkanTuerker**
Specify library name and version:  **zxing-cpp/1.0.8**

I am a user of this package and it does not build since the package version scheme of a dependency has changed.
See
https://github.com/conan-io/conan-center-index/commit/354ee84b27fd66bd89c98bbb586a3540485ca834#diff-48e3a9d98498745b6951e8e1892a7dccba4ce971fc5c4149e2b2317e40308932

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by madebr — DISMISSED** [review]

**Review by SSE4 — APPROVED** [review]

**Review by Croydon — APPROVED** [review]

**Comment by SerkanTuerker** [comment]
To one of the maintainer: could you check why the windows build is failing? The error message is not very descriptive unfortunately.

Thank you!

**Comment by SSE4** [comment]
> To one of the maintainers: could you check why the windows build is failing? The error message is not very descriptive unfortunately.
> 
> Thank you!

all I can say - test_package executable returned status/exist code 3, that's it. then we need either debug it locally, or add more logs to the test_package executable.

**Comment by SerkanTuerker** [comment]
> 

Yes, thank you, I can read as well :)  That's why I said, the error message is not very descriptive. 

The questions now are, who is responsible for this and when can we finally merge this?
I am just a user of a package which depends on stb.

I didn't know that it would take more than a month for a pull request to get merged.

**Comment by SSE4** [comment]
checking locally, it has some sort of heap corruption:
image

**Review by SSE4 — APPROVED** [review]

**Review by uilianries — APPROVED** [review]
LGTM

File: recipes/zxing-cpp/all/conanfile.py [hunk 1]
```diff
@@ -55,6 +55,15 @@ def config_options(self):
...
 
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 14)
+        try:
+            minimum_required_version = self._compiler_cpp14_support[str(self.settings.compiler)]
+            if self.settings.compiler.version < tools.Version(minimum_required_version):
+                raise ConanInvalidConfiguration("This compiler is too old. This library needs a compiler with c++14 support")
+        except KeyError:
+            self.output.warn("This recipe might not support the compiler. Consider adding it.")
+
```
[the remark this answers is not in the document]
Comment (uilianries) [reply recipes/zxing-cpp/all/conanfile.py:66]: Thank you!!!!

**Pull request merged**